algorithms/symmetry/__init__.py

```diff
@@ -373,9 +371,9 @@ def _resolution_filter(resolutionizer, min_i_mean_over_sigma_mean, min_cc_half):
     d_min_cc_half = 0
     if min_i_mean_over_sigma_mean is not None:
         try:
-            d_min_isigi = resolutionizer.resolution_i_mean_over_sigma_mean(
-                min_i_mean_over_sigma_mean
-            )
+            d_min_isigi = resolutionizer.resolution(
+                "i_mean_over_sigi_mean", limit=min_i_mean_over_sigma_mean
```

Comment:
This API documented somewhere? Assuming yes, and I have not yet hit it in the review, but I wonder if `"i_mean_over_sigi_mean"` would be better as an ersatz ENUM e.g. `resolutionizer.I_MEAN_OVER_SIGI_MEAN`?

Reply:
@graeme-winter  is f41018f the kind of thing you had in mind?

Reply:
☝️  is _precisely_ what I had in mind thank you, and will help you in the future